src/components/DeeplinkWrapper/DeeplinkRedirectLoadingIndicator.js

```diff
@@ -41,6 +41,7 @@ function DeeplinkRedirectLoadingIndicator({translate, openLinkInBrowser, session
                         width={200}
                         height={164}
                         src={Illustrations.RocketBlue}
+                        fill={undefined}
```

Comment:
Here also, set the default value for the `fill` prop in the Icon component to `undefined` instead?